ansible/roles/debops.rails_deploy/tasks/deploy_keys.yml

```diff
@@ -14,15 +14,20 @@
         rails_deploy_register_deploy_key is defined
 
 - name: Transfer the deploy key to Github
-  command: "curl --silent --header 'Authorization: token {{ rails_deploy_git_access_token }}' --data '{{ rails_deploy_key_data | to_nice_json }}' https://api.github.com/repos/{{ rails_deploy_git_account }}/{{ rails_deploy_git_repo }}/keys"
+  command: "curl --silent --header 'Authorization: token {{ rails_deploy_git_access_token }}'
+                 --data '{{ rails_deploy_key_data | to_nice_json }}'
+                 https://api.github.com/repos/{{ rails_deploy_git_account }}/{{ rails_deploy_git_repo }}/keys"
+  args:
+    warn: False
```

Comment:
It should be possible to not depend on `curl` here and do it with a Ansible module.

Reply:
This is the same reason as with the `rm` and the `command` module - my aim here was to fix the `ansible-lint` error with minimum changes to the existing code.

Reply:
Same as I explained with the `rm` and the `command` module except that I am more on your side here because this task is more complex than a simple file: state=absent. Maybe also add FIXME notes so that people working on this role the next time and test it can fix the task in one go.